{
	"document_title": "CVE-2017-2592",
	"document_type": "SUSE CVE",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "SUSE CVE-2017-2592"
		},
		"status": "Interim",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "6",
				"date": "2022-03-01T01:51:11Z",
				"description": "current"
			}
		},
		"initial_release_date": "2021-05-30T13:50:15Z",
		"current_release_date": "2022-03-01T01:51:11Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf-cve.pl",
			"date": "2020-12-27T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "CVE-2017-2592",
			"title": "CVE",
			"type": "Summary"
		},
		{
			"text": "python-oslo-middleware before versions 3.8.1, 3.19.1, 3.23.1 is vulnerable to an information disclosure. Software using the CatchError class could include sensitive values in a traceback's error message. System users could exploit this flaw to obtain sensitive information from OpenStack component error logs (for example, keystone tokens).",
			"title": "Mitre CVE Description",
			"type": "Description"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		}
	],
	"document_references": [
		{
			"type": "Self",
			"url": "https://lists.suse.com/pipermail/sle-security-updates/2017-March/002744.html",
			"description": "E-Mail link for SUSE-SU-2017:0848-1"
		},
		{
			"type": "Self",
			"url": "https://lists.suse.com/pipermail/sle-security-updates/2017-April/002817.html",
			"description": "E-Mail link for SUSE-SU-2017:1062-1"
		},
		{
			"type": "Self",
			"url": "https://lists.suse.com/pipermail/sle-security-updates/2018-November/004896.html",
			"description": "E-Mail link for SUSE-SU-2018:3924-1"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "SUSE OpenStack Cloud 6",
				"branch": [
					{
						"type": "Product Name",
						"name": "SUSE OpenStack Cloud 6",
						"full_product_name": {
							"text": "SUSE OpenStack Cloud 6",
							"product_id": "SUSE OpenStack Cloud 6",
							"cpe": "cpe:/o:suse:suse-openstack-cloud:6"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Family",
				"name": "SUSE OpenStack Cloud 7",
				"branch": [
					{
						"type": "Product Name",
						"name": "SUSE OpenStack Cloud 7",
						"full_product_name": {
							"text": "SUSE OpenStack Cloud 7",
							"product_id": "SUSE OpenStack Cloud 7",
							"cpe": "cpe:/o:suse:suse-openstack-cloud:7"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.cache-1.14.1-3.3.3",
				"full_product_name": {
					"text": "python-oslo.cache-1.14.1-3.3.3",
					"product_id": "python-oslo.cache-1.14.1-3.3.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.concurrency-3.14.1-3.3.3",
				"full_product_name": {
					"text": "python-oslo.concurrency-3.14.1-3.3.3",
					"product_id": "python-oslo.concurrency-3.14.1-3.3.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.db-4.13.6-3.3.3",
				"full_product_name": {
					"text": "python-oslo.db-4.13.6-3.3.3",
					"product_id": "python-oslo.db-4.13.6-3.3.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.log-3.16.1-3.3.3",
				"full_product_name": {
					"text": "python-oslo.log-3.16.1-3.3.3",
					"product_id": "python-oslo.log-3.16.1-3.3.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.messaging-5.10.2-3.6.3",
				"full_product_name": {
					"text": "python-oslo.messaging-5.10.2-3.6.3",
					"product_id": "python-oslo.messaging-5.10.2-3.6.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.middleware-2.8.0-3.1",
				"full_product_name": {
					"text": "python-oslo.middleware-2.8.0-3.1",
					"product_id": "python-oslo.middleware-2.8.0-3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.middleware-3.19.1-4.3.4",
				"full_product_name": {
					"text": "python-oslo.middleware-3.19.1-4.3.4",
					"product_id": "python-oslo.middleware-3.19.1-4.3.4"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.serialization-2.13.2-3.3.3",
				"full_product_name": {
					"text": "python-oslo.serialization-2.13.2-3.3.3",
					"product_id": "python-oslo.serialization-2.13.2-3.3.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.service-1.16.1-3.3.1",
				"full_product_name": {
					"text": "python-oslo.service-1.16.1-3.3.1",
					"product_id": "python-oslo.service-1.16.1-3.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.utils-3.16.1-3.3.3",
				"full_product_name": {
					"text": "python-oslo.utils-3.16.1-3.3.3",
					"product_id": "python-oslo.utils-3.16.1-3.3.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.versionedobjects-1.17.1-3.3.1",
				"full_product_name": {
					"text": "python-oslo.versionedobjects-1.17.1-3.3.1",
					"product_id": "python-oslo.versionedobjects-1.17.1-3.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslo.vmware-2.14.1-3.3.1",
				"full_product_name": {
					"text": "python-oslo.vmware-2.14.1-3.3.1",
					"product_id": "python-oslo.vmware-2.14.1-3.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python-oslotest-2.10.1-3.3.1",
				"full_product_name": {
					"text": "python-oslotest-2.10.1-3.3.1",
					"product_id": "python-oslotest-2.10.1-3.3.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python-oslo.middleware-2.8.0-3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 6"
			},
			{
				"product_reference": "python-oslo.cache-1.14.1-3.3.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.concurrency-3.14.1-3.3.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.db-4.13.6-3.3.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.log-3.16.1-3.3.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.messaging-5.10.2-3.6.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.middleware-3.19.1-4.3.4",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.serialization-2.13.2-3.3.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.service-1.16.1-3.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.utils-3.16.1-3.3.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.versionedobjects-1.17.1-3.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslo.vmware-2.14.1-3.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"product_reference": "python-oslotest-2.10.1-3.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "python-oslo-middleware before versions 3.8.1, 3.19.1, 3.23.1 is vulnerable to an information disclosure. Software using the CatchError class could include sensitive values in a traceback's error message. System users could exploit this flaw to obtain sensitive information from OpenStack component error logs (for example, keystone tokens).",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2017-2592",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"SUSE OpenStack Cloud 6:python-oslo.middleware-2.8.0-3.1",
						"SUSE OpenStack Cloud 7:python-oslo.cache-1.14.1-3.3.3",
						"SUSE OpenStack Cloud 7:python-oslo.concurrency-3.14.1-3.3.3",
						"SUSE OpenStack Cloud 7:python-oslo.db-4.13.6-3.3.3",
						"SUSE OpenStack Cloud 7:python-oslo.log-3.16.1-3.3.3",
						"SUSE OpenStack Cloud 7:python-oslo.messaging-5.10.2-3.6.3",
						"SUSE OpenStack Cloud 7:python-oslo.middleware-3.19.1-4.3.4",
						"SUSE OpenStack Cloud 7:python-oslo.serialization-2.13.2-3.3.3",
						"SUSE OpenStack Cloud 7:python-oslo.service-1.16.1-3.3.1",
						"SUSE OpenStack Cloud 7:python-oslo.utils-3.16.1-3.3.3",
						"SUSE OpenStack Cloud 7:python-oslo.versionedobjects-1.17.1-3.3.1",
						"SUSE OpenStack Cloud 7:python-oslo.vmware-2.14.1-3.3.1",
						"SUSE OpenStack Cloud 7:python-oslotest-2.10.1-3.3.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "4.3",
					"vector_v_2": "AV:N/AC:M/Au:N/C:P/I:N/A:N"
				},
				"score_set_v_3": {
					"base_score_v_3": "5.5",
					"vector_v_3": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
				}
			}
		}
	]
}
